ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManagerTest.java

```diff
@@ -75,6 +75,7 @@ public static void setupSuite() {
     fullBlockchain = BlockchainSetupUtil.forTesting().importAllBlocks();
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
I really don't like disabling the unchecked warning.  Consider using <?> when you access no templated methods.

Reply:
`@SuppressWarnings("unchecked")` is already used in this class in `shouldNotImportBlocksThatAreAlreadyBeingImported`